keyboards/nasu/info.json

```
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "Nasu",
+    "url": "https://yiancar-designs.com",
+    "maintainer": "Yiancar-Designs",
+    "width": 17.85,
+    "height": 8.9,
+    "layouts": {
+        "LAYOUT": {
```

Does not correctly key to nasu.h content
```suggestion
        "LAYOUT_alice_split_bs": {
```